.ci/openshift-ci/cluster/install_kata.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This script installs the built kata-containers in the test cluster,
+# and configure a runtime.
+
+scripts_dir=$(dirname $0)
+deployments_dir=${scripts_dir}/deployments
+configs_dir=${scripts_dir}/configs
+
+source ${scripts_dir}/../../lib.sh
+
+# Set to 'yes' if you want to configure SELinux to permissive on the cluster
+# workers.
+#
+SELINUX_PERMISSIVE=${SELINUX_PERMISSIVE:-no}
+
+# Wait all worker nodes reboot.
+#
+# Params:
+#   $1 - timeout in seconds (default to 1800).
+#
+wait_for_reboot() {
+	local delta="${1:-1800}"
+	declare -A BOOTIDS
+	local workers=($(oc get nodes | \
+		awk '{if ($3 == "worker") { print $1 } }'))
+	# Get the boot ID to compared it changed over time.
+	for node in ${workers[@]}; do
+		BOOTIDS[$node]=$(oc get -o jsonpath='{.status.nodeInfo.bootID}'\
+			node/$node)
+		echo "Wait $node reboot"
+	done
+
+	timer_start=$(date +%s)
+	while [ ${#workers[@]} -gt 0 ]; do
+		for i in ${!workers[@]}; do
+			current_id=$(oc get \
+				-o jsonpath='{.status.nodeInfo.bootID}' \
+				node/${workers[i]})
+			if [ "$current_id" != ${BOOTIDS[${workers[i]}]} ]; then
+				echo "${workers[i]} rebooted"
+				unset workers[i]
+			fi
+		done
+		if [ $(($timer_start + $delta)) -lt $(date +%s) ]; then
+			echo "Timeout: not all workers rebooted"
+			return 1
+		fi
+		sleep 10
+	done
+}
+
+oc project default
+
+worker_nodes=$(oc get nodes |  awk '{if ($3 == "worker") { print $1 } }')
+num_nodes=$(echo $worker_nodes | wc -w)
+[ $num_nodes -ne 0 ] || \
+	die "No worker nodes detected. Something is wrong with the cluster"
+
```

Comment:
Worth logging the number of nodes to help debug any issues?

Reply:
It is not needed. On line 65 (https://github.com/kata-containers/tests/pull/3139/commits/82ec5abe1e877a325640b630d59055eb44d527c6#diff-0e9f34ff52b582c56c2f0f81b858ceddd1d9ee77ae0c4b21ebb6a532e05c0184R64) it will loop through all nodes to print their names and set labels.